src/utils/log.js

```diff
@@ -0,0 +1,10 @@
+function warn(content) {
+  if (process.env.NODE_ENV !== 'production') {
+    const message = (typeof content === 'function') ? content() : content;
+    if (message) {
+      console.warn(message); // eslint-disable-line no-console
+    }
+  }
+}
+
+export default { warn }
```

Comment:
Yeah, I thought I had other logs apart from `warn`, so I structured it to have `export default { warn, error, log }`. Not sure if we'll ever have them..